test/test_settings.c

```diff
@@ -0,0 +1,124 @@
+﻿#include <LCUI_Build.h>
+#include <LCUI/LCUI.h>
+#include <LCUI/settings.h>
+#include <LCUI/main.h>
+#include <LCUI/timer.h>
+#include "test.h"
+#include "libtest.h"
+
+static int settings_change_count = 0;
+
+static void on_settings_change(LCUI_SysEvent object, void *data)
+{
+	++settings_change_count;
+}
+
+static void check_settings_frame_rate_cap(void *arg)
+{
+	char str[256];
+	int fps_limit = *((int *)arg);
+	int fps = LCUI_GetFrameCount();
+
+	sprintf(str, "should work when frame cap is %d (actual %d)", fps_limit,
+		fps);
+	it_b(str, fps <= fps_limit && fps > fps_limit - 2, TRUE);
+	LCUI_Destroy();
```

Comment:
Use `LCUI_Quit()` instead of `LCUI_Destroy()`.

Some data is still used in `LCUI_Main()`, you should not call `LCUI_Destroy()` directly to destroy them, so you should call `LCUI_Quit()`  to notify the main loop to exit and then destroy the data.

![image](https://user-images.githubusercontent.com/1730073/86309546-97e41680-bc4e-11ea-9424-5d2301deac48.png)